plugins/new-relic-dashboard/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-new-relic-dashboard",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^0.9.6",
+    "@backstage/core-components": "^0.7.2",
+    "@backstage/core-plugin-api": "^0.2.0",
+    "@backstage/plugin-catalog-react": "^0.6.3",
+    "@backstage/theme": "^0.2.12",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.57",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^6.0.2",
```

Comment:
You'll need to sync your dependencies with what the other packages in the repo have. Specifically, this version of react-router-dom is not compatible with the one that's used. I'd have expected an empty yarn.lock file in this pull request.

Reply:
@freben 
This plugin is different from the existing New Relic plugin. The existing plugin shows APM data of all accounts on a single page whereas this plugin imports dashboard from New Relic and renders it in the Entity Page

Secondly, the proxy config used in the existing plugin routes to `/v2` , which is not compatible with the endpoints we want to use. 

![Screenshot 2021-11-26 at 1 17 24 PM](https://user-images.githubusercontent.com/42934221/143545801-97e9ad37-54d6-4935-812d-79cacef576ca.png)

My suggestion would be to accept the configuration in the new plugin and later make a change in the existing plugin.

Reply:
I will do a version bump and commit an empty yarn.lock 👍

Reply:
Just to build on top of that a bit, I'm wondering if it would be reasonable to add these new features to the existing plugin instead? A plugin can export a mix of a couple of different extensions and one doesn't necessarily need to import and use all of them in an app.

Reply:
@Rugvip Yes, it's possible to do so and does make sense. But, both plugins are solving different purposes, this one purely focuses on New Relic Dashboards, the other one lists APM data. We do have different plugins from the same SaaS products in backstage. 
It's more of an AD, you'll let me know, how we should move forward.

Reply:
Alright, seems that it might be fine to move forward with separate plugins. It'll be possible to merge them together in the future too if we feel there's a need.

So with that I think it's best if we keep them as compatible as possible, in case they share any config or things like that. One of those things would be that I think it's best if we call this one `@backstage/plugin-newrelic-dashboard`, so that the naming is aligned with the existing one. Looking at their GitHub org they actually use that pattern anyway https://github.com/newrelic :grin:

Reply:
Oh, yes. That would be a good fix. Will do it. Thanks for spotting it!

Reply:
@Rugvip Did the change , renamed it to `@backstage/plugin-newrelic-dashboard`